.github/workflows/build-test.yml

```diff
@@ -0,0 +1,111 @@
+name: build-test
+# Only run the workflow when pushed to master, dev, or on pull requests.
+on: 
+  push:
+    branches: 
+      - master
+      - dev
+  pull_request:
+    branches:
+      - '**'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    environment: Default
+    steps:
+      # This should automatically cancel any previous workflows that are still
+      # running when for example new commits are pushed.
+      - uses: technote-space/auto-cancel-redundant-workflow@v1
+      - uses: actions/checkout@v2
+      - name: Cache SBT dependencies modules
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.sbt
+            ~/.ivy2/cache
+            ~/.cache/coursier
+          key: ${{ runner.os }}-build-vercors-sbt-jdk11
+      - uses: actions/setup-java@v1
+        with:
+          java-version: "11"
+      # Currently sbt and fakeroot are installed by default
+      - run: sbt debian:packageBin
+      - name: Archive VerCors .deb
+        uses: actions/upload-artifact@v2
+        with:
+          name: vercors-debian-package
+          retention-days: 1
+          path: |
+            target/*.deb
+      - name: SonarCloud Scan
+        uses: sonarsource/sonarcloud-github-action@master
+        env:
+          # This is set automatically by github
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          # SONAR_TOKEN is set in the secrets section of the github repo
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+    
+      # If this is the dev branch we also want to build the .txz binary for the release
+      - name: Build mac binary
+        if: ${{ github.ref == 'refs/heads/dev' }}
+        run: sbt universal:packageXzTarball
```

Comment:
ik wil hier trouwens liever gewoon .tar.gz van maken of wat minder aggressief compressen; deze stap duurt volgens mij erg lang.

Reply:
Gewoon "packageZipTarball" maar doen dan?

Reply:
heeft ie dan niet dezelfde naam als voor windows?

Reply:
oh alle runners staan erin of niet haha